assets/sites.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+// phpcs:ignoreFile
+
+$sites = [];
+
+// Add in the host name to sites if it starts with `bnf-`. This way we don't
+// have to worry about what TLD people use (docker or local), nor the composer
+// project name (dpl-cms per default, but could be anything).
+$host = $_SERVER['HTTP_HOST'];
+if (preg_match('/^bnf-/', $host)) {
+  $sites[$host] = 'bnf';
+}
```

Comment:
I think this prefix rule should be documented somewhere so people know it.

Reply:
Ah, yes, but where?